vmdb/app/controllers/api_controller/accounts.rb

```diff
@@ -4,7 +4,7 @@ module Accounts
     # Accounts Subcollection Supporting Methods
     #
     def accounts_query_resource(object)
-      object.send("accounts")
+      object.accounts
```

Comment:
I assume the pattern to research this would be:

```bash
grep -r 'has.*:#{method}\b'
grep -r 'def.*\b#{method}\b'
```

Seems a git show of this PR + grep could nail down the methods and then generate greps.